packages/notification/src/features/utils.ts

```diff
@@ -117,7 +117,7 @@ export function getPersonName(
   if (!name) {
     error(record, `name not found in patient resource for ${compositionCode}`)
   }
-  return [name.given?.join(' '), name.family].join(' ').trim()
+  return [name.given?.join(' ').trim(), name.family].join(' ').trim()
```

Comment:
@tahmidrahman-dsi how did the extra space ended up in the name in the first place? Are we missing a trim somewhere where input is collected?

Reply:
Understood 👍 Can you add a comment there explaining this please, so we wont forget?

Reply:
Sorry, I am going to add the comment in my next PR